logexp/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file builds an image for the log exporter tool. For more info,
+# have a look at the tool's README.md file.
+
+FROM gcr.io/google-containers/kubekins-test:v20160822
```

Comment:
kubekins-test has a bunch of stuff you don't need, I suspect. maybe just use `python-slim` (which removes the need to install python), or use `gcr.io/google-containers/debian-base-amd64:0.1` (along with `clean-install python`)